### ISSUE TITLE
App aspect ration overlaps.

### ISSUE THREAD

**Issue by Flam3z** [TeamNewPipe/NewPipe#4442]
### Checklist

- [x] I am using the latest version. 
- [x] I checked, but didn't find any duplicates of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one bug. I will open one issue for every bug report I want to file.

### Steps to reproduce the bug

1.Open any video in portrait mode.
2.switch to landscape.
3.minimize the video in portrait mode.

### Actual behaviour

The aspect ratio overlaps and im unable to see my on screen nav buttons unless i swipe them to show.

### Expected behavior

Aspect ratio within the app should stay the same.
![Imagepipe_3](https://user-images.githubusercontent.com/57234536/95411155-ba439b00-08f3-11eb-8903-7961ece93883.jpg)

### Device info

 - Android version/Custom ROM version: Android 10
 - Device model: Galaxy S10

**Comment by opusforlife2** [comment]
You say you encounter the issue in landscape, but your screenshot is portrait.

By the way, did you think the 3rd item in the checklist was optional?

**Comment by Flam3z** [comment]
@opusforlife2 yes i thought it was optional, my apologies, the bug is encountered on both portrait and landscape. I am able to reproduce the bug 7/10 times.

**Comment by opusforlife2** [comment]
Your 3rd step says "minimise the video" but there is no minimised player in your screenshot.

Do what the check box says and tick it, then?

**Comment by Flam3z** [comment]
@opusforlife2 the bug seems to occur when i lock my phone and it automatically plays in the background, then when i unlock, my on screen buttons are forced to hide unless i swipe them up for a bit which is what makes newpipe take up my whlle screen. Kind of hard to capture that in a screenshot im sorry.

**Comment by opusforlife2** [comment]
Screen record is also an option.

**Comment by Flam3z** [comment]
@opusforlife2 any reliable site to send you the recording? Or try and send it as it is

**Comment by opusforlife2** [comment]
Github is pretty reliable. 👍  Zip it up, or better, gzip it up, and upload it.

**Comment by Flam3z** [comment]
[Uploading NP bug recording.zip…]()

**Comment by Flam3z** [comment]
Im having issues with uploading it to github, Errors on my end.

**Comment by opusforlife2** [comment]
You're probably blocking the domain needed in uBO. Try to upload, then when you get the error and the domain shows in the panel, unblock it.

**Comment by opusforlife2** [comment]
@Flam3z Fixed in the latest version?

**Comment by Flam3z** [comment]
@opusforlife2 sadly not lol, I just realized I had created the same bug report a couple hours ago and decided to close this one as I had finally found a reliable way to send clips of visual bugs. (To my defense I was very tired at the time of making the new bug report.)
